scripts/shell.nix

```diff
@@ -2,8 +2,28 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-{ pkgs ? import (import ../nix/nix/sources.nix {}).nixpkgs {} }:
+let
+  overlays =
+    [
+      (self: super: {
+        gh = (super.callPackage "${super.path}/pkgs/applications/version-management/git-and-tools/gh" {
+          buildGoModule = args: super.buildGoModule (args // rec {
+            version = "1.2.0";
+            vendorSha256 = "0ybbwbw4vdsxdq4w75s1i0dqad844sfgs69b3vlscwfm6g3i9h51";
+            src = self.fetchFromGitHub {
+              owner = "cli";
+              repo = "cli";
+              rev = "v${version}";
+              sha256 = "17hbgi1jh4p07r4p5mr7w7p01i6zzr28mn5i4jaki7p0jwfqbvvi";
+            };
+          });
```

Comment:
Just to note, in general it's better to push/cherry-pick updates to packages like this into our nixpkgs fork. A cherry-picked commit would even be automatically overlayed on our next nixpkgs rebase.